### PR TITLE
Remove the PIN UI component

### PR DESCRIPTION
**Pull request by suitcasemedia**
Happy to do this differently if i have misunderstood

### CODE REVIEW

File: client/src/components/LinkDisplay/index.js [hunk 1]
```diff
@@ -54,12 +54,7 @@ const LinkDisplay = ({ content }) => {
           </button>
         </div>
       </div>
-      <div
-        className={`${styles.pinDisplay} ${darkMode ? styles.darkOpenLink : styles.lightOpenLink}`}
-      >
-        <span className={styles.pinValidMsg}>PIN (valid for 30 minutes)</span>
-        <PinDisplay content={content.pin} />
```
Comment (muke1908) [inline client/src/components/LinkDisplay/index.js:61]: You can also remove `PinDisplay` from import

**Review by muke1908 — CHANGES_REQUESTED** [review]

File: client/src/components/LinkDisplay/index.js (hunk 1 above)
Comment (suitcasemedia) [reply client/src/components/LinkDisplay/index.js:61]: No problem, done.

Comment (suitcasemedia) [reply client/src/components/LinkDisplay/index.js:61]: I can also delete the PinDisplay component folder

Comment (muke1908) [reply client/src/components/LinkDisplay/index.js:61]: lets leave it for now

**Pull request merged**